### ISSUE TITLE
[Bug]: Update Facebook SDK to the latest version

### ISSUE THREAD

**Issue by jakublebiedzinski** [oddbit/flutter_facebook_app_events#308]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Regarding the nullability of Facebook SDK, your package should support the latest version of Facebook SDK (at least 16.1.3). This version contains an important fix for linking, logging and launching events within the app. Here is a link to that fix: https://github.com/facebook/facebook-ios-sdk/issues/2209

### Expected Behavior

Facebook SDK should not crash the app when clicking on Live Activity

### Steps To Reproduce

1. Create Live Activity
2. Click on it

### Platform

iOS

### Flutter Doctor

```shell
[✓] Flutter (Channel stable, 3.10.6, on macOS 13.4.1 22F770820d darwin-arm64, locale en-PL)
    • Flutter version 3.10.6 on channel stable at /Users/xxxxx/Developer/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f468f3366c (7 days ago), 2023-07-12 15:19:05 -0700
    • Engine revision cdbeda788a
    • Dart version 3.0.6
    • DevTools version 2.23.1

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
    • Android SDK at /Users/xxxxxx/Library/Android/sdk
    • Platform android-33, build-tools 33.0.1
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b802.4-9586694)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.11.3

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Android Studio (version 2022.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b802.4-9586694)

[✓] VS Code (version 1.80.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension can be installed from:
      🔨 https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (2 available)
    • M2102J20SG (mobile) • d35e495d • android-arm64 • Android 13 (API 33)
    • macOS (desktop)     • macos    • darwin-arm64  • macOS 13.4.1 22F770820d darwin-arm64

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

**Comment by antonoddbit** [comment]
Hi @jakublebiedzinski thanks for the input, we will check on using latest FacebookSDK

**Comment by chhs2131** [comment]
Is this error relevant to the issue?

```python
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In Podfile:
    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 16.0)

Specs satisfying the `FBSDKCoreKit (~> 16.0)` dependency were found, but they required a higher minimum deployment target.
```
(when I send `pod install` to terminal)

**Comment by iballan** [comment]
> Is this error relevant to the issue?
> 
> ```python
> [!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
>   In Podfile:
>     facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
>       FBSDKCoreKit (~> 16.0)
> 
> Specs satisfying the `FBSDKCoreKit (~> 16.0)` dependency were found, but they required a higher minimum deployment target.
> ```
> 
> (when I send `pod install` to terminal)

I think it is related to upgrading the minimum deployment target to iOS 13.0

**Comment by antonoddbit** [comment]
@jakublebiedzinski @vscmaster thanks for the patience, the patch should be coming shortly
